src/pybind/rados/rados.pyx

```diff
@@ -351,80 +351,125 @@ class Error(Exception):
         return (self.__class__, (self.message, self.errno))
 
 class InvalidArgumentError(Error):
-    pass
+    def __init__(self, message, errno=None):
+        super(InvalidArgumentError, self).__init__(
+            ("RADOS Invalid Argument error (%s)" % message), errno)
```

Comment:
i'd suggest remove the "error" in the error message. as it's always an error in this context.

Reply:
@tchaikov I'll change it